helmfile.d/values/grafana/grafana-dashboards.yaml.gotmpl

```
@@ -52,7 +52,19 @@ notDeveloperVisible:
 
 {{- if .Values.objectStorage.sync.enabled }}
 sync:
+  {{- $buckets := (list) }}
   {{- range .Values.objectStorage.sync.buckets }}
-  - {{ .source }}
+  {{- if .source }}
+  {{- $buckets = append $buckets .source }}
+  {{- end }}
+  {{- end }}
+  {{- if .Values.objectStorage.sync.syncDefaultBuckets }}
+  {{- range .Values.objectStorage.buckets }}
+  {{- $buckets = append $buckets . }}
+  {{- end }}
+  {{- $buckets = $buckets | uniq }}
+  {{- end }}
+  {{- range $buckets }}
```

```suggestion
  {{- end }}
  {{- range $buckets | uniq | sortAlpha }}
```
Not sure _if_ it would be unstable if it isn't sorted, but we can with this ensure it won't.
Also, great find!